src/Microsoft.Framework.Runtime/ProjectResolver.cs

```diff
@@ -49,7 +49,7 @@ public bool TryResolveProject(string name, out Project project)
 
         private void Initialize(string projectPath, string rootPath)
         {
-            _searchPaths.Add(Path.GetDirectoryName(projectPath));
+            _searchPaths.Add(new DirectoryInfo(projectPath).Parent.FullName);
```

Comment:
Path.GetDirectoryName("this/is/a/path") returns "this/is/a"
Path.GetDirectoryName("this/is/a/path/") returns "this/is/a/path"
Path.GetFullPath() doesn't remove trailing slash for you